cdhweb/people/models.py

```diff
@@ -286,6 +287,12 @@ def current_title(self):
         if current_positions.exists():
             return current_positions.first().title
 
+    @property
+    def most_recent_title(self):
+        """Return the most recent of any titles held by this Person."""
+        if self.positions.exists():
+            return self.positions.first().title
```

Comment:
IDK how much it matters, but I think this will run two queries when we only need one. Slightly better to use `first()` instead of `exists()` and then if the first result is not none return the title attribute.